build/ios/mojom/cpp_transformations.h

```diff
@@ -223,7 +223,7 @@ NS_INLINE NSDictionary<NSString*, NSString*>* NSDictionaryFromMap(
 
 /// Convert a String to String mapping to an NSDictionary
 NS_INLINE NSDictionary<NSString*, NSString*>* NSDictionaryFromMap(
-    base::flat_map<std::string, std::string> m) {
+    const base::flat_map<std::string, std::string>& m) {
   const auto d = [NSMutableDictionary new];
   if (m.empty()) {
     return @{};
```

Comment:
Should iterations over `m` below also use ref  `for (auto& item : m)` ?

Reply:
Good catch. Fixed.